package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "clean": "rimraf lib dist",
     "lint": "eslint src test",
-    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
-    "build:umd": "cross-env BABEL_ENV=commonjs NODE_ENV=development webpack src/index.js dist/flypro.js",
-    "build:umd:min": "cross-env BABEL_ENV=commonjs NODE_ENV=production webpack src/index.js dist/flypro.min.js",
+    "build:commonjs": "babel src --out-dir lib",
+    "build:umd": "umd -c Flypro lib/index.js dist/flypro.js",
```

Comment:
stripped env vars, got rid of webpack.
Why to use bundler on only one file?
Just wrap with umd already built with babel file